### PR TITLE
inpost tracking

### LINKED ISSUE #49: inpost tracking

### PR DESCRIPTION
**Pull request by hkubus**

### CODE REVIEW

File: src/cmds/inpost.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        return message.reply(
+            client.language.get(message.guild.preferredLocale, 'INPOST_ARGS')
+        );
+    }
+    let statusURL;
+    if (message.guild.preferredLocale === 'pl') {
+        statusURL = 'https://api-shipx-pl.easypack24.net/v1/statuses';
+    } else {
+        statusURL = 'https://api-shipx-pl.easypack24.net/v1/statuses?lang=en';
+    }
```
Comment (SuperchupuDev) [inline src/cmds/inpost.ts]: can we make this a string in the locales json?

File: src/cmds/inpost.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    const statuses: any = await (await fetch(statusURL)).json();
```
Comment (SuperchupuDev) [inline src/cmds/inpost.ts]: dont use double await
```suggestion
    const statuses: any = await fetch(statusURL).then(res => res.json());
```

**Review by SuperchupuDev — CHANGES_REQUESTED** [review]
can you make this a slash command too?

File: src/cmds/inpost.ts (hunk 1 above)
Comment (hkubus) [reply src/cmds/inpost.ts]: we can but ummmmmmmmmmmmmmmm yeah

Comment (hkubus) [reply src/cmds/inpost.ts]: we could use ternary operators to shorten that

Comment (SuperchupuDev) [reply src/cmds/inpost.ts]: just add it as a locale string, it's easier and otherwise it will break if we ever add more languages

**Review by SuperchupuDev — APPROVED** [review]

**Pull request merged**